example/main.go

```diff
@@ -20,6 +22,26 @@ func Main() int {
 		return log.FErrf("Error opening terminal: %v", err)
 	}
 	defer t.Close()
-	fmt.Printf("Terminal is open - is valid %t\n", t.IsTerminal())
+	t.SetPrompt("Fortio> ")
+	isTerm := t.IsTerminal()
+	// t.Out will add the needed \r for each \n when term is in raw mode
+	log.SetOutput(&terminal.CRWriter{Out: os.Stderr})
+	log.Config.ForceColor = isTerm
+	log.SetColorMode()
+	fmt.Fprintf(t.Out, "Terminal is open\nis valid %t\n", isTerm)
+	l, err := t.ReadLine()
+	if err != nil {
+		if errors.Is(err, io.EOF) {
+			log.Infof("EOF received, exiting.")
+			return 0
+		}
+		return log.FErrf("Error reading line: %v", err)
+	}
```

Comment:
Easier to read and understand

```suggestion
	if errors.Is(err, io.EOF) {
		log.Infof("EOF received, exiting.")
		return 0
	}
	if err != nil {
		return log.FErrf("Error reading line: %v", err)
	}
```